src/components/views/elements/AppPermission.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import url from 'url';
+
+export default class AppPermission extends React.Component {
+    constructor(props) {
+        super(props);
+
+        const curlBase = this.getCurlBase();
+        this.state = {
+            curlBase: curlBase,
+        };
+    }
+
+    // Return string representation of content URL without query parameters
+    getCurlBase() {
+        const wurl = url.parse(this.props.url);
+        let curl;
+        let curlString;
+
+        const searchParams = new URLSearchParams(wurl.search);
+
+        if(searchParams && searchParams.get('url')) {
```

Comment:
This should not be the way we check if a URL has a `curl` or not. It's entirely valid for URLs to have `?url=` and have it not mean a `curl`. We need to check if the URL starts with the `integrations_rest_url`.